custom_targets.json

```diff
@@ -14,6 +14,11 @@
         "inherits": [
             "MCU_STM32WLe5xC"
         ],
-        "device_name": "STM32WLE5JCIx"
-    }
+        "device_name": "STM32WLE5CCUx"
+    },
+    "RAK3172_BREAKOUT": {
+        "inherits": [
+            "RAK3172"
+        ]
+    },
```

Comment:
remove this last comma

Reply:
yeah there was also the e5 typo error sorry